src/services/walkthroughServices.js

```diff
@@ -112,6 +187,15 @@ const findOrCreateOpenshiftResource = (openshiftResourceDef, resToFind, compareF
     return Promise.resolve(foundResource);
   });
 
+const replaceOpenShiftResource = (resourceDef, resourceRes, replaceIfFn = (resObj => resObj.metadata.name === resourceRes.metadata.name)) =>
+  findOrCreateOpenshiftResource(resourceDef, resourceRes)
+    .then(resource => {
+      if (replaceIfFn(resource)) {
+        return remove(resourceDef, resourceRes).then(() => create(resourceDef, resourceRes));
```

Comment:
is this what the `oc replace` command does?
Is there actually not a replace api?